setup/setup.js

```diff
@@ -100,7 +105,8 @@ const setupFunctions = {
   dataProcessing,
   populateIndex,
   sleep,
-  meiliUpdates
+  meiliUpdates,
```

Comment:
```suggestion
  watchUpdates,
```

Avoid using `meili` here and nowhere else. It is confusing as it implies that the other functions do not communicate with MeiliSearch